allegro.cpp

```diff
@@ -3466,7 +3466,7 @@ void Alg_seq::beat_to_measure(double beat, int *measure, double *m_beat,
     }
     bpm = prev.num * 4 / prev.den;
     m = m + (beat - prev.beat) / bpm;
-    *measure = m;
+    *measure = static_cast<int>(m);
```

Comment:
std::floor(m) ?

Reply:
`std::floor()` still returns a floating-point type for who knows why, so a cast will still be necessary to silence the warning. Will `m` ever be negative? otherwise, the behavior will be the same as truncating via cast.

Reply:
Keeping C-style cast here too, resolving this comment